packaging/installer/UPDATE.md

```diff
@@ -26,7 +26,7 @@ most installations, this is `/etc/netdata`.
 Use `cd` to navigate to the Netdata config directory, then use `ls -a` to look for a file called `.environment`.
 
 -   If the `.environment` file _does not_ exist, reinstall with your [package manager](#deb-or-rpm-packages).
--   If the `.environtment` file _does_ exist, check its contents with `less .environment`.
+-   If the `.environment` file _does_ exist, check its contents with `less .environment`.
```

Comment:
This is a kind of important user facing change as the current text is misleading